pkg/apis/kubeone/helpers.go

```diff
@@ -88,6 +90,17 @@ func (h *HostConfig) SetLeader(leader bool) {
 	h.IsLeader = leader
 }
 
+func (c KubeOneCluster) AddonOperatingSystemManagerEnabled() bool {
```

Comment:
Same as the comment related to the variable name, I'd drop this `Addon` prefix.

```suggestion
func (c KubeOneCluster) OperatingSystemManagerEnabled() bool {
```